addons/sourcemod/scripting/surftimer/sql.sp

```diff
@@ -3412,6 +3424,97 @@ public void SQL_selectCheckpointsCallback(Handle owner, Handle hndl, const char[
 	}
 }
 
+public void db_viewReplayCPTicks(char szMapName[128])
+{
+	char szQuery[1024];
+
+	for(int style = 0; style < MAX_STYLES; style++){
+		g_bReplayTickFound[style] = false;
+
+		Format(szQuery, sizeof(szQuery), sql_selectReplayCPTicksAll, szMapName, style);
+		SQL_TQuery(g_hDb, SQL_selectReplayCPTicksCallback, szQuery, DBPrio_Low);
```

Comment:
so btw. `DBPrio_Low` is here `data` which shouldn't be the logic here. https://sourcemod.dev/#/dbi/function.SQL_TQuery